src/frame/plane.rs

```diff
@@ -256,8 +256,8 @@ impl<T: Pixel> Plane<T> {
     let yorigin = self.cfg.yorigin;
     let stride = self.cfg.stride;
     let alloc_height = self.cfg.alloc_height;
-    let width = w >> self.cfg.xdec;
-    let height = h >> self.cfg.ydec;
+    let width = w + self.cfg.xdec >> self.cfg.xdec;
```

Comment:
Isn't that a `round_shift`?

Reply:
@lu-zero wouldn't a `round_shift` be `w + (1 << self.cfg.xdec >> 1) >> self.cfg.xdec`

Reply:
Technically the same, but also doesn't work directly since the function in `util` accepts only `i32`. If it were documented and generic it could be worth it.

Reply:
@tmatth yes, given the range of xdec it behaves the same way.